### DISCUSSION TITLE
macOS Command Line

### DISCUSSION THREAD

**Discussion by Paul-Gilmartin** [Stellarium/stellarium#3294] [General]
Chapter 6 of the User's Guide pdf discusses command line options.
Is this supposed to be available in macOS?  If not, the first paragraph
should list the systems supporting command line ise.

I discover that the shell command:
    /Applications/Stellarium.app/Contents/MacOS/stellarium -h

... seems t o work according to the Guide, but I suspect this is
accidental and shouldn't be documented.

Thanks,
gil

**Comment by 10110111** [comment]
It's a standard command-line option (`--help` and `-h` do the same), which is handled by `QCommandLineParser`. Its presence in Stellarium doesn't depend on platform.

**Comment by gzotti** [comment]
Why should that be accidential?
